api/registrations/views.py

```diff
@@ -860,6 +861,54 @@ def get_object(self):
         return obj
 
 
+class LinkedRegistrationsList(JSONAPIBaseView, generics.ListAPIView, RegistrationMixin):
```

Comment:
This looks an awful lot like the LinkedRegistrationsList in Collections. Can you DRY this up by making a base version without the RegistrationMixin nor CollectionMixin then just subclass here?